Text analytics analyzer (#292)
microsoft/presidio: presidio-analyzer/README.MD

The `analyze` method should return a list of [RecognizerResult](presidio_analyzer/recognizer_result.py). Refer to the [code documentation](presidio_analyzer/entity_recognizer.py) for more information.

b. Reference and add the new class to the `RecognizerRegistry` module, in the `load_predefined_recognizers` method, which registers all code based recognizers.

## Using Azure Text Analytics to detect PII data

> Presidio comes with a built-in `TextAnalyticsRecognizer`.

Reviewer: Please let's add more details for people that are not familiar with the service. At the least explain that we're using the NER capability of the service, provide a link to https://azure.microsoft.com/en-us/services/cognitive-services/text-analytics/ and https://docs.microsoft.com/en-us/azure/cognitive-services/text-analytics/how-tos/text-analytics-how-to-entity-linking?tabs=version-3#named-entity-recognition-versions-and-features and direct developers to the portal page where they can find the endpoint and key.

Reviewer: I can make these changes if you like :)

Author: @CodeRunRepeat  please do

Reviewer: Done, @itye-msft please check it out.

The `analyze` method should return a list of [RecognizerResult](presidio_analyzer/recognizer_result.py). Refer to the [code documentation](presidio_analyzer/entity_recognizer.py) for more information.

b. Reference and add the new class to the `RecognizerRegistry` module, in the `load_predefined_recognizers` method, which registers all code based recognizers.

## Using Azure Text Analytics to detect PII data

The [Azure Text Analytics service](https://azure.microsoft.com/en-us/services/cognitive-services/text-analytics/)

> provides with some PII detection features in its [3.0-preview version](https://docs.microsoft.com/en-us/azure/cognitive-services/text-analytics/how-tos/text-analytics-how-to-entity-linking?tabs=version-3#named-entity-recognition-versions-and-features) (as of February 2020).

Reviewer 2: ```suggestion
offers new PII detection features in its [3.0-preview version](https://docs.microsoft.com/en-us/azure/cognitive-services/text-analytics/how-tos/text-analytics-how-to-entity-linking?tabs=version-3#named-entity-recognition-versions-and-features) (as of April 2020).
```

> Presidio comes with a built-in `TextAnalyticsRecognizer` that integrates with the Azure Text Analytics service

Reviewer 3: potential recognizer?

Reviewer: Wdym? :)

if you want to leverage these capabilities.
When you choose to use this recognizer, please set the following environment variables required to connect to the service:

```bash

> $ TEXT_ANALYTICS_KEY=<your-text-analytics-key>

Reviewer 2: What about the new `ENABLE_TEXT_ANALYTICS_RECOGNIZER` var?
